core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1138,6 +1139,9 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
     case "BITWISE_XOR":
       unparseBitwiseFunctions(writer, call, XOR, leftPrec, rightPrec);
       break;
+    case "BITNOT":
+      unparseBitNotFunction(writer, call, leftPrec, rightPrec);
```

Comment:
leftPrec and rightPrec can be removed from here.

Reply:
Done